packages/tasks/src/tasks/image-to-video/data.ts

```diff
@@ -0,0 +1,172 @@
+import type { TaskDataCustom } from "../index.js";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "A dataset of images and short video clips for image-to-video generation research.",
+			id: "some/image-to-video-dataset",
+		},
+		{
+			description: "A benchmark dataset for reference-based video generation.",
+			id: "ali-vilab/VACE-Benchmark",
+		},
+		{
+			description: "A dataset for scoring video generation styles.",
+			id: "Rapidata/sora-video-generation-style-likert-scoring",
+		},
+		{
+			description: "A dataset for captioned videos ",
+			id: "BestWishYsh/ChronoMagic",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "image-to-video-input.jpg",
+				type: "img",
+			},
+			{
+				label: "Optional prompt",
+				content: "This penguin is dancing",
+				type: "text",
+			},
+		],
+		outputs: [
+			{
+				filename: "image-to-video-output.gif",
+				type: "img",
+			},
+		],
+	},
+	metrics: [
+		{
+			description:
+				"Fréchet Video Distance (FVD) measures the perceptual similarity between the distributions of generated videos and a set of real videos, assessing overall visual quality and temporal coherence of the video generated from an input image.",
+			id: "fvd",
+		},
+		{
+			description:
+				"CLIP Score measures the semantic similarity between a textual prompt (if provided alongside the input image) and the generated video frames. It evaluates how well the video's generated content and motion align with the textual description, conditioned on the initial image.",
+			id: "clip_score",
+		},
+		{
+			description:
+				"First Frame Fidelity, often measured using LPIPS (Learned Perceptual Image Patch Similarity), PSNR, or SSIM, quantifies how closely the first frame of the generated video matches the input conditioning image.",
+			id: "lpips",
+		},
+		{
+			description:
+				"Identity Preservation Score measures the consistency of identity (e.g., a person's face or a specific object's characteristics) between the input image and throughout the generated video frames, often calculated using features from specialized models like face recognition (e.g., ArcFace) or re-identification models.",
+			id: "identity_preservation",
+		},
+		{
+			description:
+				"Motion Score evaluates the quality, realism, and temporal consistency of motion in the video generated from a static image. This can be based on optical flow analysis (e.g., smoothness, magnitude), consistency of object trajectories, or specific motion plausibility assessments.",
+			id: "motion_score",
+		},
+	],
+	models: [
+		{
+			description: "LTX-Video, a 13B parameter model for high quality video generation",
+			id: "Lightricks/LTX-Video-0.9.7-dev",
+		},
+		{
+			description: "A 1.3B parameter model for reference-based video generation",
```

Comment:
fyi we add very few models, and simple explanations on them (this one is ok but others are a bit too long